addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -196,7 +196,7 @@
 <template id="header" name="Forum Index">
     <t t-if="forum.active" t-call="website_forum.layout">
         <div class="oe_structure" id="oe_structure_website_forum_header_1"/>
-        <div id="wrap" t-attf-class="o_wforum_wrapper position-relative container d-flex d-xl-grid px-lg-0 #{website_forum_action}">
+        <div id="wrap" t-attf-class="o_wforum_wrapper position-relative container d-flex d-xl-grid px-lg-0 #{forum_welcome_message and 'h-100'} #{website_forum_action}">
```

Comment:
Let's not have `False` as a class:
```suggestion
        <div id="wrap" t-attf-class="o_wforum_wrapper position-relative container d-flex d-xl-grid px-lg-0 #{'h-100' if forum_welcome_message else ''} #{website_forum_action}">
```

Reply:
I've done this for all the `and` in classes :)

Once you've checked, I can squash my commits together

Reply:
Check! Thank you!